contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_step_3.handlebars

```diff
@@ -0,0 +1,42 @@
+<div id="clipboard" class="pull-right">
```

Comment:
The header section that remains the same should be extracted and turned into a separate template to avoid repetition.